js/everything.js

```diff
@@ -580,32 +573,32 @@ $(function() {
     // How do we know if the value is truly invalid?
     // Preload the form from the URL
     var currentVideoID = getCurrentVideoID();
-    if (currentVideoID) {
+    if (currentVideoID)
+    {
         $("#v").attr("value", currentVideoID);
                             // get similar videos, populate playList
-        $.ajax({
-            url: "https://www.googleapis.com/youtube/v3/search",
-            dataType: "json",
-            async: false,
-            data: {
-                key: youTubeDataApiKey,
-                part: "snippet",
-                type: "video",
-                relatedToVideoId: currentVideoID
-            },
-            success: function(data) {
-				// push items into playlist
-                for (var i = 0;i < data.items.length;i ++ )
-				{
-                    playList.push(data.items[i].id.videoId);
+        if (!isFileProtocol()) {
+            $.ajax({
+                url: "https://www.googleapis.com/youtube/v3/search",
+                dataType: "json",
+                async: false,
+                data: {
+                    key: youTubeDataApiKey,
+                    part: "snippet",
+                    type: "video",
+                    relatedToVideoId: currentVideoID
+                },
+                success: function(data) {
+                    // push items into playlist
+                    for (var i = 0;i < data.items.length;i ++ )
```

Comment:
The spacing is inconsistent around operators. Look at what's in the code vs. what's in my comment